plugins/woocommerce/tests/e2e-pw/utils/editor.js

```diff
@@ -50,18 +50,33 @@ const goToPostEditor = async ( { page } ) => {
 };
 
 const fillPageTitle = async ( page, title ) => {
-	await ( await getCanvas( page ) ).getByLabel( 'Add title' ).click();
-	await ( await getCanvas( page ) ).getByLabel( 'Add title' ).fill( title );
+	// TODO (Gutenberg 19.9): Keep only the "Block: Title" label locator.
+	// Current stable version of Gutenberg (19.7) uses the "Add title" label locator.
+	// Upcoming Gutenberg 19.9 uses the "Block: Title" one. We should use it instead when GB 19.9 comes out.
+	const canvas = await getCanvas( page );
+	const block_title = canvas
+		.getByLabel( 'Add title' )
+		.or( canvas.getByLabel( 'Block: Title' ) );
+	await block_title.click();
+	await block_title.fill( title );
 };
 
 const insertBlock = async ( page, blockName, wpVersion = null ) => {
+	// Focus on "Empty block" element before inserting a new block.
+	// Otherwise Gutenberg nightly (v19.9-nightly) would display "{Block name} can't be inserted."
+	const emptyBlock = ( await getCanvas( page ) ).getByLabel( 'Empty block' );
+	if ( await emptyBlock.isVisible() ) {
+		await emptyBlock.click();
+	}
```

Comment:
In GB stable (v19.7) the "Empty block" doesn't appear right away when you start with a blank page. Instead there's the "Add default block", and you have to click on it first in order to show the "Empty block" element.

GB nightly (v19.9-nightly) on the other hand doesn't show the "Add default block" anymore. It shows the "Empty block" right away. So we need this if condition to handle both situations.